src/Migration/MigrationRunner.cs

```diff
@@ -99,7 +99,9 @@ protected virtual IMigrationContext GetContext()
         /// </summary>
         public virtual async Task DowngradeAsync()
         {
-            await Task.FromException(new NotImplementedException("Downgrade has not been implemented yet. Provide your input in https://github.com/rapidcore/issues/issues/14"));
```

Comment:
It might be nicer to conditionally add FromException as an extension method on `Task`.